checks/runtimeCheck.go

```diff
@@ -37,9 +43,11 @@ func latestAgentTag(r *runtimeConfig) error {
 	if err != nil {
 		return err
 	}
-	err = json.Unmarshal(body, &r)
-	if err != nil {
-		return err
+	var re = regexp.MustCompile(`\/releases\/tag\/(v[0-9.]+)`)
```

Comment:
A small nit: compiling a regular expression is best done once, rather than inline, since it's often more expensive than actually applying one. It's common to have your regular expressions as private static variables in your module.